docs/source/developers/guide/step_by_step/finding_issues.rst

```diff
@@ -29,4 +29,90 @@
 Finding good first issues 🔎
-****************************
\ No newline at end of file
+****************************
+
+You have successfully built the Arrow library; congrats!
+
+The next step is finding something to work on. As mentioned before,
+you might already have a bug to fix in mind, or a new feature that you want to implement.
+Or you still need an issue to work on and you need some help with finding one.
+
+For both cases, JIRA is the issue tracker that we use.
+
+First we will explain how to use JIRA if you have a fix or a feature to work on that doesn't yet have a JIRA ticket open, in which case you will need to create a JIRA ticket yourself.
+
+Secondly, we will show you a way to find good first issues to work on.
+
+
+Creating a JIRA account
+==========================
+
+First thing you need to do is to make an account on the ASF JIRA following
+`this link <https://issues.apache.org/jira/secure/Signup!default.jspa>`_. You will be asked
+to select a language and choose an avatar if you wish. If the registration is successful
+you will see:
+
+.. figure:: jira_new_account.jpeg
+   :scale: 70 %
+   :alt: creating an ASF JIRA account
+
+   The window you get after creating an account on the ASF JIRA.
+
+
+How to create a JIRA issue
+==========================
+
+After creating an account you can click **Create an issue** and select **Apache Arrow project**
+and **Type** of the issue (Bug, Feature, …).
+
+.. figure:: jira_create_issue.jpeg
+   :scale: 70 %
+   :alt: creating JIRA issue
+
+   The window to create a JIRA issue.
+
+If you are already in JIRA dashboard click the red ``create`` button in the top to do the same.
+
+You are ready to create the issue! Add a title and a description following the
+:ref:`tips for using JIRA <jira-tips>` and you are ready to go!
+
+.. seealso::
+	:ref:`Tips for using JIRA <jira-tips>`
+
+You don’t need any special permissions on JIRA to be able to create issues.
+Once you are more involved in the project and want to do more on JIRA, for example assigning
+yourself an issue, you will need **“Contributor” permissions**. To get this role, ask on the
+:ref:`mailing_list` or in the comment of the JIRA issue you created.
+
+When the ticket is created you can start a discussion about it in the JIRA comments section.
+
+How we use JIRA to find an issue
+================================
+
+To make it easier for you to find issues that are well-suited for new contributors, we have added labels
+like “good-first-issue” or “beginner” to some JIRA tickets.
+
+.. seealso::
+	Search for good first/second issues with labels like in the `link here
+	<https://issues.apache.org/jira/browse/ARROW-14659?filter=-4&jql=project%20%3D%20ARROW%20AND%20status%20%3D%20Open%20AND%20labels%20in%20(Beginner%2C%20beginner%2C%20beginners%2C%20beginnner%2C%20beginner-friendly%2C%20good-first-issue%2C%20good-second-issue%2C%20GoodForNewContributors%2C%20newbie%2C%20easyfix%2C%20documentation)%20order%20by%20created%20DESC>`_
```

Comment:
```suggestion
	<https://issues.apache.org/jira/issues/?filter=-4&jql=project%20%3D%20ARROW%20AND%20status%20%3D%20Open%20AND%20labels%20in%20(Beginner%2C%20beginner%2C%20beginners%2C%20beginnner%2C%20beginner-friendly%2C%20good-first-issue%2C%20good-second-issue%2C%20GoodForNewContributors%2C%20newbie%2C%20easyfix%2C%20documentation)%20order%20by%20created%20DESC>`_
```

(the current link goes to a specific issue; JIRA is really annoying / confusing on this aspect ...)